### ISSUE TITLE
JDBC plugin not supports unsigned int?

### ISSUE THREAD

**Issue by jsvd** [logstash-plugins/logstash-input-jdbc#169]
Migrated from: https://github.com/elastic/logstash/issues/5997 (by @rayenyang)

I have a table which has a unsigned int column in MySQL.Its a pity that exception occurred when I used Logstash to read this table:

```
**e[33mException when executing JDBC query {:exception=>#<Sequel::DatabaseError: J
ava::ComMysqlJdbcExceptionsJdbc4::MySQLDataException: '2.926689519E9' in column
'2' is outside valid range for the datatype INTEGER.>, :level=>:warn}e**
```

my lostash config:

```
input{
jdbc{jdbc_driver_library=>"mysql-connector-java-5.1.37.jar" 
jdbc_driver_class=>"com.mysql.jdbc.Driver" 
jdbc_connection_string=>"jdbc:mysql://localhost:3306/mydb
jdbc_user=>"root" 
jdbc_password=>"1234" 
schedule=>"* * * * *" 
jdbc_paging_enabled=>true 
jdbc_page_size=>1000 
statement=>"SELECT * from mytable where record_id>:sql_last_value order by record_id desc" 
use_column_value=>true 
tracking_column=>"record_id" 
}
}
output{
elasticsearch{

    }
}
```

**Comment by 00sivan** [comment]
bump. JDBC input assumes that even unsigned ints are signed. I've had to build views in MySQL to get around it using "casts" in the statements. Would be great to have this working.

**Comment by jakelandis** [comment]
This was fixed today via [sequel 5.2](https://github.com/jeremyevans/sequel/blob/master/CHANGELOG). 

Updating (or re-installing the existing plugin) should pull in the latest sequel version with this fix.

`bin/logstash-plugin install --version "4.3.0" logstash-input-jdbc`
